tsdb/series_index_test.go

```diff
@@ -6,9 +6,14 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/influxdb/models"
 	"github.com/influxdata/influxdb/tsdb"
 )
 
+func typedSeriesIDHelper(id uint64) tsdb.SeriesIDTyped {
+	return tsdb.NewSeriesID(id).WithType(models.Empty)
+}
```

Comment:
Can this be called `toSeriesIDTyped()`? "helper" doesn't explain what it does very well IMO.